docsrc/communicate/js-from-fable.md

```diff
@@ -61,6 +61,28 @@ If the value is globally accessible in JS, you can use the `Global` attribute wi
  let [<Global>] console: JS.Console = jsNative
 ```
 
+
+#### OOP Class definition and inheritance
+
+Assuming we need to import an ES6 class or even a typescript class from which we only know the types, it is advised to attacj it to a class with an abstract member defining the types and a default implementation which points to the jsNative reference:
```

Comment:
There is a typoe at: `attacj `

I think adding jsNative  betwing quotes can help better understand that's it's code like that `jsNative`